### ISSUE TITLE
Teamspeak fails to start

### ISSUE THREAD

**Issue by doogie544** [flathub/com.teamspeak.TeamSpeak#14]
After the last flatpak update to teamspeak  the app will not longer start.

I have tried several things:

1. Removed all flatpak installations (only had TS3 and it's required dependencies)
2 Reinstalled teamspeak and dependencies
3. Tested the Teamspeak official download and checked the start.sh against the flatpak version- they were identical and the official version started normally
4. Started Teamspeak from the .sh in the flatpak directory /var/lib/..../extra it started normally and worked perfectly

I have to assume that some config in the flatpak version is not correct.  

I get the following when trying to start it with `flatpak run com.teamspeak.TeamSpeak` 

`flatpak run com.teamspeak.TeamSpeak 
QCoreApplication::applicationDirPath: Please instantiate the QApplication object first
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (8 8 8 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 8 8 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 8 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (8 8 8 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 8 8 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 8 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 0)
qt.glx: qglx_findConfig: Failed to finding matching FBConfig (1 1 1 0)
Could not initialize GLX
/app/extra/ts3client_runscript.sh: line 47:     6 Aborted                 (core dumped) ./ts3client_linux_amd64 $@`

My flatpak install:

`flatpak list
Name                    Application ID                        Version   Branch   Installation
TeamSpeak               com.teamspeak.TeamSpeak               3.3.2     stable   system
Freedesktop Platform    org.freedesktop.Platform              19.08.6   19.08    system
default                 org.freedesktop.Platform.GL.default             19.08    system
openh264                org.freedesktop.Platform.openh264               19.08    system
Breeze Gtk theme        org.gtk.Gtk3theme.Breeze-Dark                   3.22     system
`

Other info:

I am using Ubuntu 19.10 KDE
Nvidia 2070 Super 
Nvidia drivers 440.48

Edit: for some reason the code black it not working correctly, it looks formatted correctly but it's still showing up incorrectly when posted

**Comment by Zlopez** [comment]
This looks like issue with NVidia, you are missing runtime extension for your graphics card.
Try running `flatpak update` from terminal manually.

**Comment by doogie544** [comment]
Yeah I had tried that before but it was finding nothing.  Just tried it again and it shows a Nvidia update so we will see if it works.

I had tried to manually install the newest nvidia driver available before I posted the bug report but it didn't work.  Maybe flathub was behind on the nvida drivers

BTW I always do updates manually and installs for that matter so I can track them.  I only use Discover to search for them and notify me that there are updates, using the software manger for flatpaks fails too often.
